frontend/src/actions/users.ts

```diff
@@ -79,22 +92,31 @@ export const followUser = (userId: number, token: string) => {
         }
       );
 
+      callback();
+
       dispatch<FollowUserAction>({
         type: ActionTypes.followUser,
-        payload: userId,
+        payload: user_id,
       });
     });
   };
 };
 
-export const unfollowUser = (userId: number, token: string) => {
-  return (dispatch: Dispatch) => {
+export const unfollowUser = (data: {
+  user_id: number;
+  token: string;
+  callback: Function;
+}) => {
+  return async (dispatch: Dispatch) => {
     backend.get('/sanctum/csrf-cookie').then(async (csrf_response) => {
+      let user_id, token, callback;
+      ({ user_id, token, callback } = data);
```

Comment:
Can be done in single line
```suggestion
      let { user_id, token, callback } = data;
```

Reply:
done